src/plugins/touch/touch.js

```diff
@@ -36,6 +37,7 @@
          lastX = x;
 
          window.impress().swipe( diff / window.innerWidth );
+         event.preventDefault();
```

Comment:
This one seems more defensible, since it is intended that any swipe = touchmove should end up in this plugin and nowhere else.